src/current/v23.1/molt-verify.md

```diff
@@ -78,19 +77,27 @@ The following databases are currently supported:
 
 Flag | Description
 ----------|------------
-`--concurrency int` | Number of shards to process at a time. <br>Default value: 16 <br>For faster verification, set this flag to a higher value. <br>Note: Table splitting by shard only works for `int`, `uuid`, and `float` data types.
-`--table_splits int` | Number of shards to split the table into. <br>Default value: 16
-`--row_batch_size int` | Number of rows to get from a table at a time. <br>Default value: 20000
+`--source` | (Required) Connection string for the source database.
+`--target` | (Required) Connection string for the target database.
+`--concurrency` | Number of shards to process at a time. <br>**Default:** 16 <br>For faster verification, set this flag to a higher value. <br>Note: Table splitting by shard only works for [`INT`]({% link {{ page.version.version }}/int.md %}), [`UUID`]({% link {{ page.version.version }}/uuid.md %}), and [`FLOAT`]({% link {{ page.version.version }}/float.md %}) data types.
+`--row-batch-size` | Number of rows to get from a table at a time. <br>**Default:** 20000
+`--table-filter` | Verify tables that match a specified [regular expression](https://wikipedia.org/wiki/Regular_expression).
+`--schema-filter` | Verify schemas that match a specified [regular expression](https://wikipedia.org/wiki/Regular_expression).
+`--continuous` | Verify tables in a continuous loop. <br />**Default:** `false`
+`--live` | Retry verification on rows before emitting warnings or errors. This is useful during live data import, when temporary mismatches can occur. <br />**Default:** `false`
 
 ## Limitations
 
-- While verifying data, MOLT Verify pages 20,000 rows at a time by default, and row values can change in between, which can lead to temporary inconsistencies in data. You can change the row batch size using the `--row_batch_size int` [flag](#supported-flags).
+- While verifying data, MOLT Verify pages 20,000 rows at a time by default, and row values can change in between, which can lead to temporary inconsistencies in data. You can change the row batch size using the `--row_batch_size` [flag](#supported-flags).
 - MySQL enums and set types are not supported.
-- When a `STRING` is used as a [primary key]({% link {{ page.version.version }}/primary-key.md %}), MOLT Verify may generate additional warnings due to differences in how CockroachDB and other databases handle case sensitivity in strings.
+- When a [`STRING`]({% link {{ page.version.version }}/string.md %}) is used as a [primary key]({% link {{ page.version.version }}/primary-key.md %}), validation may fail to differences in how CockroachDB and other databases handle case sensitivity in strings.
```

Comment:
we now check for collation mismatches in `molt verify` and fail to validate if we mismatch.

Reply:
I'm trying to understand how that should change the description here. So it looks like MySQL default collation is `utf8mb4_0900_ai_ci`. Does this imply that you will get a collation mismatch whenever a collation isn't explicitly set? Does that specifically apply to string PK columns?

Reply:
Yeah if we detect on crdb you don't have a Unicode related collation on the column we don't allow you to proceed.